% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
  11pt,
]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  pdftitle={Coding-complete RNA virus genomes assembled from murine cecal metatranscriptomes},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage[margin=1.0in]{geometry}
\usepackage{graphicx,grffile}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\usepackage{helvet} % Helvetica font
\renewcommand*\familydefault{\sfdefault} % Use the sans serif version of the font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}

\title{\textbf{Coding-complete RNA virus genomes assembled from murine cecal
metatranscriptomes}}
\author{}
\date{\vspace{-2.5em}}

\begin{document}
\maketitle

\vspace{35mm}

\textbf{Running title:} RNA Viral Genomes from Murine Metatranscriptomes

\vspace{35mm}

Joshua M.A.~Stough, Andrew J. Beaudoin, Patrick D.
Schloss\textsuperscript{\(\dagger\)}

\vspace{40mm}

\(\dagger\) To whom correspondence should be addressed:
\href{mailto:pschloss@umich.edu}{\nolinkurl{pschloss@umich.edu}}

Department of Microbiology and Immunology, University of Michigan, Ann
Arbor, MI 48109

\newpage
\linenumbers

\hypertarget{abstract}{%
\subsection{Abstract}\label{abstract}}

Efforts to catalog viral diversity in the gut microbiome have largely
focused on DNA viruses, while RNA viruses remain understudied. To
address this, we screened assemblies of previously published mouse gut
metatranscriptomes for the presence of RNA viruses. We identified the
coding-complete genomes of an Astrovirus and 5 Mitovirus-like viruses.

\newpage

The viral fraction of the mammalian gut microbiome forms a crucial
component in the relationship between microbe and host. Bacterial
viruses serve as an important source of genetic diversity and population
control for the microbiota, driving its ecology and evolution (1).
Mammalian viruses disrupt the gut environment through infection and the
response of the host immune system (2). Bacterial and mammalian viruses
make significant contributions to host health and disease. Current
efforts to describe the diversity of viruses present in the gut have
focused on using shotgun metagenomics to identify double-stranded DNA
viruses, predominantly bacteriophage and host pathogens (3). However,
this method ignores viruses with RNA genomes, which make up a
considerable portion of the environmental viromes (4).

We re-analyzed deeply-sequenced metatranscriptome data produced by our
lab for the study of microbiome dynamics in a mouse model of
\emph{Clostridioides difficile} infection (5, 6). Briefly, C57Bl/6 mice
from a breeding colony we maintain at the University of Michigan were
treated with one of three different antibiotics (clindamycin,
streptomycin, or cefoperazone). After a 24 hour recovery period, the
mice were infected with \emph{C. difficile} strain 630. Germ free
C57Bl/6 mice were also mono-associated with \emph{C. difficile} strain
630. Cecal contents were removed from each animal 18 hours post
infection and frozen for RNA extraction and sequencing. RNA sequences
from each sample were trimmed of adapter sequences and low-quality bases
using Trimmomatic v0.39 and assembled individually using rnaSPAdes
v3.13.1 (7) and concatenated for dereplication, resulting in 70,779
contigs longer than 1 kb. Contigs were screened for the presence of RdRP
coding sequences using Blast v2.9.0 against a database containing all
Viral RefSeq protein sequences annotated as RdRP (screening database
available online as described below) with a maximum e-value of
10\textsuperscript{-20}, resulting in 29 contigs. RdRP is conserved
amongst almost all RNA viruses without a DNA stage in genome
replication. These contigs were then annotated with Interproscan
v5.39-77.0 (8, 9). We constructed phylogenetic trees from RdRP protein
sequences using IQ-TREE v1.6.12 (10).

Two classes of RNA viruses were assembled with high coverage with
sequences originating from most of the mouse treatment groups, including
germ-free mice. First, a 6,811 base-long astrovirus genome (GC 56.6\%)
was obtained with 1,683.5-fold coverage (Figure 1A). The genome
contained 3 predicted open reading frames encoding a capsid, RdRP, and a
trypsin-like peptidase, and appears to be closely related to Murine
Astrovirus in \emph{Astroviridae}. Second, 5 distinct, but closely
related RNA virus genomes, designated Putative Mitovirus JS1 through
JS5, ranging in length from 2,309 to 2,447 bases with 4.6 to
16,078.8-fold coverage and average GC content of 46.2\% belonged to a
previously undescribed clade of \emph{Narnaviridae} adjacent to the
Mitoviruses (Figure 1B). These RNA virus genomes will facilitate future
studies of RNA virus biology in the murine microbiome.

\textbf{Data Availability.} The RNA-seq data are available the NCBI
Sequence Read Archive (SRA) database under the accession numbers
\href{https://www.ncbi.nlm.nih.gov/bioproject/354635}{PRJNA354635}
(\emph{C. difficile} infected mice) and
\href{https://www.ncbi.nlm.nih.gov/bioproject/415307}{PRJNA415307}
(mock-infected mice). The assembled genomes are available at the
National Center for Biotechnology Information (NCBI) GenBank under the
accession numbers
\href{https://www.ncbi.nlm.nih.gov/nuccore/?term=MN780842+MN780843+MN780844+MN780845+MN780846+MN780847}{MN780842-MN780847}.
All of the scripts and software used to perform this analysis are
available at
\url{https://github.com/SchlossLab/Stough_Mouse_RNA_Virome_MRA_2019}.

\hypertarget{acknowledgements}{%
\subsection{Acknowledgements}\label{acknowledgements}}

This research was supported by NIH grant U01AI12455. The funders had no
role in study design, data collection and interpretation, or the
decision to submit the work for publication.

\newpage

\hypertarget{references}{%
\subsection{References}\label{references}}

\hypertarget{refs}{}
\leavevmode\hypertarget{ref-ogilvie_human_2015}{}%
1. \textbf{Ogilvie LA}, \textbf{Jones BV}. 2015. The human gut virome: A
multifaceted majority. Frontiers in Microbiology \textbf{6}:918.
doi:\href{https://doi.org/10.3389/fmicb.2015.00918}{10.3389/fmicb.2015.00918}.

\leavevmode\hypertarget{ref-legoff_eukaryotic_2017}{}%
2. \textbf{Legoff J}, \textbf{Resche-Rigon M}, \textbf{Bouquet J},
\textbf{Robin M}, \textbf{Naccache SN}, \textbf{Mercier-Delarue S},
\textbf{Federman S}, \textbf{Samayoa E}, \textbf{Rousseau C},
\textbf{Piron P}, \textbf{Kapel N}, \textbf{Simon F}, \textbf{Soci√© G},
\textbf{Chiu CY}. 2017. The eukaryotic gut virome in hematopoietic stem
cell transplantation: New clues in enteric graft-versus-host disease.
Nature Medicine \textbf{23}:1080--1085.
doi:\href{https://doi.org/10.1038/nm.4380}{10.1038/nm.4380}.

\leavevmode\hypertarget{ref-garmaeva_studying_2019}{}%
3. \textbf{Garmaeva S}, \textbf{Sinha T}, \textbf{Kurilshikov A},
\textbf{Fu J}, \textbf{Wijmenga C}, \textbf{Zhernakova A}. 2019.
Studying the gut virome in the metagenomic era: Challenges and
perspectives. BMC Biology \textbf{17}:84.
doi:\href{https://doi.org/10.1186/s12915-019-0704-y}{10.1186/s12915-019-0704-y}.

\leavevmode\hypertarget{ref-culley_new_2018}{}%
4. \textbf{Culley A}. 2018. New insight into the RNA aquatic virosphere
via viromics. Virus Research \textbf{244}:84--89.
doi:\href{https://doi.org/10.1016/j.virusres.2017.11.008}{10.1016/j.virusres.2017.11.008}.

\leavevmode\hypertarget{ref-jenior_clostridium_2017}{}%
5. \textbf{Jenior ML}, \textbf{Leslie JL}, \textbf{Young VB},
\textbf{Schloss PD}. 2017. \emph{Clostridium difficile} colonizes
alternative nutrient niches during infection across distinct murine gut
microbiomes. mSystems \textbf{2}:e00063--17.
doi:\href{https://doi.org/10.1128/mSystems.00063-17}{10.1128/mSystems.00063-17}.

\leavevmode\hypertarget{ref-jenior_clostridium_2018}{}%
6. \textbf{Jenior ML}, \textbf{Leslie JL}, \textbf{Young VB},
\textbf{Schloss PD}. 2018. \emph{Clostridium difficile} alters the
structure and metabolism of distinct cecal microbiomes during initial
infection to promote sustained colonization. mSphere
\textbf{3}:e00261--18.
doi:\href{https://doi.org/10.1128/mSphere.00261-18}{10.1128/mSphere.00261-18}.

\leavevmode\hypertarget{ref-bankevich_spades:_2012}{}%
7. \textbf{Bankevich A}, \textbf{Nurk S}, \textbf{Antipov D},
\textbf{Gurevich AA}, \textbf{Dvorkin M}, \textbf{Kulikov AS},
\textbf{Lesin VM}, \textbf{Nikolenko SI}, \textbf{Pham S},
\textbf{Prjibelski AD}, \textbf{Pyshkin AV}, \textbf{Sirotkin AV},
\textbf{Vyahhi N}, \textbf{Tesler G}, \textbf{Alekseyev MA},
\textbf{Pevzner PA}. 2012. SPAdes: A new genome assembly algorithm and
its applications to single-cell sequencing. Journal of Computational
Biology: A Journal of Computational Molecular Cell Biology
\textbf{19}:455--477.
doi:\href{https://doi.org/10.1089/cmb.2012.0021}{10.1089/cmb.2012.0021}.

\leavevmode\hypertarget{ref-hoang_ufboot2:_2018}{}%
8. UFBoot2: Improving the ultrafast bootstrap approximation.

\leavevmode\hypertarget{ref-kalyaanamoorthy_modelfinder:_2017}{}%
9. \textbf{Kalyaanamoorthy S}, \textbf{Minh BQ}, \textbf{Wong TKF},
\textbf{Haeseler A von}, \textbf{Jermiin LS}. 2017. ModelFinder: Fast
model selection for accurate phylogenetic estimates. Nature Methods
\textbf{14}:587--589.
doi:\href{https://doi.org/10.1038/nmeth.4285}{10.1038/nmeth.4285}.

\leavevmode\hypertarget{ref-nguyen_iq-tree:_2015}{}%
10. \textbf{Nguyen L-T}, \textbf{Schmidt HA}, \textbf{Haeseler A von},
\textbf{Minh BQ}. 2015. IQ-TREE: A fast and effective stochastic
algorithm for estimating maximum-likelihood phylogenies. Molecular
Biology and Evolution \textbf{32}:268--274.
doi:\href{https://doi.org/10.1093/molbev/msu300}{10.1093/molbev/msu300}.

\newpage

\textbf{Figure 1. Phylogenetic trees showing the relatives of the
metatranscriptome assembled genomes.} Maximum Likelihood phylogenetic
trees constructed from RdRP amino acid sequences for (A) Astroviruses
and (B) Narnaviruses. Node annotations represent IQTree Ultra-fast
Bootstrap statistics, values less than 50\% were excluded from the tree.
Scale bars are marked in red to the left of each tree. Highlight colors
in (B) represent major Narnavirus taxa: Orange - Ourmiaviruses, Pink -
Ourmia-like Mycoviruses, Gray - Narnaviruses, Blue - Mitoviruses, Purple
- Murine Mitovirus-like viruses, Green - Leviviruses.

\end{document}
