---
title: "**Complete RNA Virus Genomes Assembled from Murine Cecal Metatranscriptomes**"
bibliography: references.bib
output:
  pdf_document:
    keep_tex: true
    includes:
      in_header: header.tex
csl: american-society-for-microbiology.csl
fontsize: 11pt
geometry: margin=1.0in
---


```{r knitr_settings, eval=TRUE, echo=FALSE, cache=FALSE}
options(tidyverse.quiet = TRUE)

library(tidyverse)
library(knitr)
library(tinytex)

opts_chunk$set("tidy" = TRUE)
opts_chunk$set("echo" = FALSE)
opts_chunk$set("eval" = TRUE)
opts_chunk$set("warning" = FALSE)
opts_chunk$set("cache" = FALSE)

inline_hook <- function(x){
	print(x)

	if(is.list(x)){
		x <- unlist(x)
	}

	if(is.numeric(x)){
		if(abs(x - round(x)) < .Machine$double.eps^0.5){
			paste(format(x,big.mark=',', digits=0, scientific=FALSE))
		} else {
			paste(format(x,big.mark=',', digits=1, nsmall=1, scientific=FALSE))
		}
	} else {
    	paste(x)      
	}
}
knitr::knit_hooks$set(inline=inline_hook)
```

``` {r LoadData, eval=TRUE, echo=FALSE, cache=FALSE, warning=FALSE, message=FALSE}
source("../code/contig_stats.R")
```


\vspace{35mm}

**Running title:** RNA Viral Genomes from Murine Metatranscriptomes

\vspace{35mm}


Joshua M.A. Stough, Andrew Beaudoin, Patrick D. Schloss^$\dagger$^

\vspace{40mm}

$\dagger$ To whom correspondence should be addressed: pschloss@umich.edu

Department of Microbiology and Immunology, University of Michigan, Ann Arbor, MI 48109


\newpage
\linenumbers


## Abstract

Efforts to catalogue viral diversity in the gut microbiome have largely ignored RNA viruses. To address this, we screened assemblies of previously published mouse gut metatranscriptomes for the presence of RNA viruses. We identified the complete genomes of a Astrovirus and 5 Mitovirus-like viruses.

\newpage

The viral fraction of the mammalian gut microbiome forms a crucial component in the relationship between microbe and host. Bacterial viruses serve as an important source of genetic diversity and population control for the microbiota, driving its ecology and evolution [@ogilvie_human_2015]. Mammalian viruses disrupt the gut environment through infection and the response of the host immune system [@legoff_eukaryotic_2017]. Bacterial and mammalian viruses make significant contributions to host health and disease. Current efforts to describe the diversity of viruses present in the gut have focused on using shotgun metagenomics to identify double-stranded DNA viruses, predominantly bacteriophage and host pathogens [@garmaeva_studying_2019]. However, this method ignores viruses with RNA genomes, which make up a considerable portion of the environmental viromes [@culley_new_2018].

We re-analyzed deeply-sequenced metatranscriptome data produced by our lab for the study of microbiome dynamics in a mouse model for *Clostridioides difficile* infection [@jenior_clostridium_2017; @jenior_clostridium_2018]. Briefly, C57Bl/6 mice from a breeding colony we maintain at the University of Michigan were treated with one of three different antibiotics (clindamycin, streptomycin, or cefoperazone). After a 24 hour recovery period, the mice were infected with *C. difficile* strain 630. Cecal contents were removed from each animal 18 hours post infection and frozen for RNA extraction and sequencing. RNA sequences from each sample were assembled individually using rnaSPAdes v3.13.1 [@bankevich_spades:_2012] and concatenated for dereplication, resulting in `r num_scaffolds` contigs longer than 1 kb. Contigs were then screened against a custom RefSeq database of viral RNA-dependent RNA polymerase (RdRP) protein sequences with a maximum e-value of 10^-20^, resulting in `r num_hits` contig hits. RdRP is conserved amongst almost all RNA viruses without a DNA stage in genome replication. These contigs were then annotated with Interproscan v5.39-77.0 [@hoang_ufboot2:_2018; @kalyaanamoorthy_modelfinder:_2017]. We constructed phylogenetic trees from RdRP protein sequences using IQ-TREE v1.6.12 [@nguyen_iq-tree:_2015].

Two classes of RNA viruses were assembled with high coverage with sequences originating from most of the mouse treatment groups, including germ-free mice. First, a `r astro_length` base-long astrovirus genome (GC `r astro_gc`%) was obtained with `r astro_coverage`-fold coverage (Figure 1A). The genome contained 3 predicted open reading frames encoding a capsid, RdRP, and a trypsin-like peptidase. Second, 5 distinct, but closely related RNA virus genomes ranging in length from `r mito_length$low` to `r mito_length$high` bases with `r mito_cov$low` to `r mito_cov$high`-fold coverage and average GC content of `r mito_gc` belonged to a previously undescribed clade of Narnaviridae adjacent to the Mitoviruses (Figure 1B). These RNA virus genomes will facilitate future studies of RNA virus biology in the murine microbiome.


**Data Availability.** The RNA-seq data are available the NCBI Sequence Read Archive (SRA) database under the accession numbers [PRJNA354635](https://www.ncbi.nlm.nih.gov/bioproject/354635) (*C. difficile* infected mice) and [PRJNA415307](https://www.ncbi.nlm.nih.gov/bioproject/415307) (mock-infected mice). The assembled genomes are available at the National Center for Biotechnology Information (NCBI) GenBank under the accession numbers [MN780842-MN780847](). All of the scripts and software used to perform this analysis are available at https://github.com/JMAStough/Stough_Mouse_RNA_Virome_MRA_2019.

## Acknowledgements
This research was supported by NIH grant U01AI12455. The funders had no role in study design, data collection and interpretation, or the decision to submit the work for publication.

\newpage

## References
<div id="refs"></div>

\newpage


**Figure 1. Phylogenetic trees showing the relatives of the metatranscriptome assembled genomes.** Maximum Likelihood phylogenetic trees constructed from RdRP amino acid sequences for (A) Astroviruses and (B) Narnaviruses. Node annotations represent IQTree Ultra-fast Bootstrap statistics, values less than 50% were excluded from the tree. Scale bars are marked in red to the left of each tree. Highlight colors in (B) represent major Narnavirus taxa: Orange - Ourmiaviruses, Pink - Ourmia-like Mycoviruses, Gray - Narnaviruses, Blue - Mitoviruses, Purple - Murine Mitovirus-like viruses, Green - Leviviruses.
