\documentclass[11pt,]{article}
\usepackage{lmodern}
\usepackage{amssymb,amsmath}
\usepackage{ifxetex,ifluatex}
\usepackage{fixltx2e} % provides \textsubscript
\ifnum 0\ifxetex 1\fi\ifluatex 1\fi=0 % if pdftex
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
\else % if luatex or xelatex
  \ifxetex
    \usepackage{mathspec}
  \else
    \usepackage{fontspec}
  \fi
  \defaultfontfeatures{Ligatures=TeX,Scale=MatchLowercase}
\fi
% use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
% use microtype if available
\IfFileExists{microtype.sty}{%
\usepackage{microtype}
\UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\usepackage[margin=1.0in]{geometry}
\usepackage{hyperref}
\hypersetup{unicode=true,
            pdftitle={Complete RNA Virus Genomes Assembled from Murine Cecal Metatranscriptomes},
            pdfborder={0 0 0},
            breaklinks=true}
\urlstyle{same}  % don't use monospace font for urls
\usepackage{graphicx}
% grffile has become a legacy package: https://ctan.org/pkg/grffile
\IfFileExists{grffile.sty}{%
\usepackage{grffile}
}{}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
\IfFileExists{parskip.sty}{%
\usepackage{parskip}
}{% else
\setlength{\parindent}{0pt}
\setlength{\parskip}{6pt plus 2pt minus 1pt}
}
\setlength{\emergencystretch}{3em}  % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{0}
% Redefines (sub)paragraphs to behave more like sections
\ifx\paragraph\undefined\else
\let\oldparagraph\paragraph
\renewcommand{\paragraph}[1]{\oldparagraph{#1}\mbox{}}
\fi
\ifx\subparagraph\undefined\else
\let\oldsubparagraph\subparagraph
\renewcommand{\subparagraph}[1]{\oldsubparagraph{#1}\mbox{}}
\fi

%%% Use protect on footnotes to avoid problems with footnotes in titles
\let\rmarkdownfootnote\footnote%
\def\footnote{\protect\rmarkdownfootnote}

%%% Change title format to be more compact
\usepackage{titling}

% Create subtitle command for use in maketitle
\providecommand{\subtitle}[1]{
  \posttitle{
    \begin{center}\large#1\end{center}
    }
}

\setlength{\droptitle}{-2em}

  \title{\textbf{Complete RNA Virus Genomes Assembled from Murine Cecal
Metatranscriptomes}}
    \pretitle{\vspace{\droptitle}\centering\huge}
  \posttitle{\par}
    \author{}
    \preauthor{}\postauthor{}
    \date{}
    \predate{}\postdate{}
  
\usepackage{helvet} % Helvetica font
\renewcommand*\familydefault{\sfdefault} % Use the sans serif version of the font
\usepackage[T1]{fontenc}

\usepackage[none]{hyphenat}

\usepackage{setspace}
\doublespacing
\setlength{\parskip}{1em}

\usepackage{lineno}

\usepackage{pdfpages}

\begin{document}
\maketitle

\vspace{35mm}

\textbf{Running title:} RNA Viral Genomes from Murine Metatranscriptomes

\vspace{35mm}

Joshua M.A.~Stough, Andrew Beaudoin, Patrick D.
Schloss\textsuperscript{\(\dagger\)}

\vspace{40mm}

\(\dagger\) To whom correspondence should be addressed:
\href{mailto:pschloss@umich.edu}{\nolinkurl{pschloss@umich.edu}}

Department of Microbiology and Immunology, University of Michigan, Ann
Arbor, MI 48109

\newpage
\linenumbers

\hypertarget{abstract}{%
\subsection{Abstract}\label{abstract}}

Efforts to catalogue viral diversity in the gut microbiome have largely
ignored RNA viruses. To address this, we screened assemblies of
previously published mouse gut metatranscriptomes for the presence of
RNA viruses. We identified the complete genomes of a Astrovirus and 5
Mitovirus-like viruses.

\newpage

The viral fraction of the mammalian gut microbiome forms a crucial
component in the relationship between microbe and host. Bacterial
viruses serve as an important source of genetic diversity and population
control for the microbiota, driving its ecology and evolution (1).
Mammalian viruses disrupt the gut environment through infection and the
response of the host immune system (2). Bacterial and mammalian viruses
make significant contributions to host health and disease. Current
efforts to describe the diversity of viruses present in the gut have
focused on using shotgun metagenomics to identify double-stranded DNA
viruses, predominantly bacteriophage and host pathogens
{[}\textbf{INSERT REFERENCE}{]}. However, this method ignores viruses
with RNA genomes, which make up a considerable portion of the
environmental viromes (3).

We re-analyzed deeply-sequenced metatranscriptome data produced by our
lab for the study of microbiome dynamics in a mouse model for
\emph{Clostridioides difficile} infection (4, 5). Briefly, C57Bl/6 mice
from a breeding colony we maintain at the University of Michigan were
treated with one of three different antibiotics (clindamycin,
streptomycin, or cefoperazone). After a 24 hour recovery period, the
mice were infected with \emph{C. difficile} strain 630. Cecal contents
were removed from each animal 18 hours post infection and frozen for RNA
extraction and sequencing. RNA sequences from each sample were assembled
individually using rnaSPAdes v3.13.1 (6) and concatenated for
dereplication, resulting in \textbf{R 70,779} contigs longer than 1 kb.
Contigs were then screened against a custom RefSeq database of viral
RNA-dependent RNA polymerase (RdRP) protein sequences with a maximum
e-value of 10\textsuperscript{-20}, resulting in \textbf{R 22} contig
hits. RdRP is conserved amongst almost all RNA viruses without a DNA
stage in genome replication. These contigs were then annotated with
Interproscan v5.39-77.0 (7, 8). We constructed phylogenetic trees from
RdRP protein sequences using IQ-TREE v1.6.12 (9).

Two classes of RNA viruses were assembled with high coverage with
sequences originating from most of the mouse treatment groups, including
germ-free mice. First, a \textbf{R 6811} base-long astrovirus genome was
obtained with \textbf{R 1683}-fold coverage (Figure 1A). The genome
contained 3 predicted open reading frames encoding a capsid, RdRP, and a
trypsin-like peptidase. Second, 5 distinct, but closely related RNA
virus genomes ranging in length from \textbf{R 2309 to 2447} bases with
\textbf{R XXXXX} to \textbf{R XXXXXX}-fold coverage belonged to a
previously undescribed clade of Narnaviridae adjacent to the Mitoviruses
(Figure 1B). These RNA virus genomes will facilitate future studies of
RNA virus biology in the murine microbiome.

\textbf{Data Availability.} The RNA-seq data are available the NCBI
Sequence Read Archive (SRA) database under the accession numbers
\href{https://www.ncbi.nlm.nih.gov/bioproject/354635}{PRJNA354635}
(\emph{C. difficile} infected mice) and
\href{https://www.ncbi.nlm.nih.gov/bioproject/415307}{PRJNA415307}
(mock-infected mice). The assembled genomes are available at the
National Center for Biotechnology Information (NCBI) GenBank under the
accession numbers \href{}{MN780842-MN780847}. All of the scripts and
software used to perform this analysis are available at
\url{https://github.com/JMAStough/Stough_Mouse_RNA_Virome_MRA_2019}.

\hypertarget{acknowledgements}{%
\subsection{Acknowledgements}\label{acknowledgements}}

This research was supported by NIH grant U01AI12455. The funders had no
role in study design, data collection and interpretation, or the
decision to submit the work for publication.

\newpage

\hypertarget{references}{%
\subsection{References}\label{references}}

\hypertarget{refs}{}
\leavevmode\hypertarget{ref-ogilvie_human_2015}{}%
1. Ogilvie LA, Jones BV. 2015. The human gut virome: A multifaceted
majority. Frontiers in Microbiology 6.

\leavevmode\hypertarget{ref-legoff_eukaryotic_2017}{}%
2. Legoff J, Resche-Rigon M, Bouquet J, Robin M, Naccache SN,
Mercier-Delarue S, Federman S, Samayoa E, Rousseau C, Piron P, Kapel N,
Simon F, Soci√© G, Chiu CY. 2017. The eukaryotic gut virome in
hematopoietic stem cell transplantation: New clues in enteric
graft-versus-host disease. Nature Medicine 23:1080--1085.

\leavevmode\hypertarget{ref-culley_new_2018}{}%
3. Culley A. 2018. New insight into the RNA aquatic virosphere via
viromics. Virus Research 244:84--89.

\leavevmode\hypertarget{ref-jenior_clostridium_2017}{}%
4. Jenior ML, Leslie JL, Young VB, Schloss PD. 2017. Clostridium
difficile Colonizes Alternative Nutrient Niches during Infection across
Distinct Murine Gut Microbiomes. mSystems 2.

\leavevmode\hypertarget{ref-jenior_clostridium_2018}{}%
5. Jenior ML, Leslie JL, Young VB, Schloss PD. 2018. Clostridium
difficile Alters the Structure and Metabolism of Distinct Cecal
Microbiomes during Initial Infection To Promote Sustained Colonization.
mSphere 3.

\leavevmode\hypertarget{ref-bankevich_spades:_2012}{}%
6. Bankevich A, Nurk S, Antipov D, Gurevich AA, Dvorkin M, Kulikov AS,
Lesin VM, Nikolenko SI, Pham S, Prjibelski AD, Pyshkin AV, Sirotkin AV,
Vyahhi N, Tesler G, Alekseyev MA, Pevzner PA. 2012. SPAdes: A new genome
assembly algorithm and its applications to single-cell sequencing.
Journal of Computational Biology: A Journal of Computational Molecular
Cell Biology 19:455--477.

\leavevmode\hypertarget{ref-hoang_ufboot2:_2018}{}%
7. Hoang DT, Chernomor O, Haeseler A von, Minh BQ, Vinh LS. 2018.
UFBoot2: Improving the Ultrafast Bootstrap Approximation. Molecular
Biology and Evolution 35:518--522.

\leavevmode\hypertarget{ref-kalyaanamoorthy_modelfinder:_2017}{}%
8. Kalyaanamoorthy S, Minh BQ, Wong TKF, Haeseler A von, Jermiin LS.
2017. ModelFinder: Fast model selection for accurate phylogenetic
estimates. Nature Methods 14:587--589.

\leavevmode\hypertarget{ref-nguyen_iq-tree:_2015}{}%
9. Nguyen L-T, Schmidt HA, Haeseler A von, Minh BQ. 2015. IQ-TREE: A
Fast and Effective Stochastic Algorithm for Estimating
Maximum-Likelihood Phylogenies. Molecular Biology and Evolution
32:268--274.

\newpage

\textbf{Figure 1. Phylogenetic trees showing the relatives of the
metatranscriptome assembled genomes.} Maximum Likelihood phylogenetic
trees constructed from RdRP amino acid sequences for (A) Astroviruses
and (B) Narnaviruses. Node annotations represent IQTree Ultra-fast
Bootstrap statistics, values less than 50\% were excluded from the tree.
Scale bars are marked in red to the left of each tree. Highlight colors
in (B) represent major Narnavirus taxa: Orange - Ourmiaviruses, Pink -
Ourmia-like Mycoviruses, Gray - Narnaviruses, Blue - Mitoviruses, Purple
- Murine Mitovirus-like viruses, Green - Leviviruses.


\end{document}
